src/OSPSuite.Presentation/Services/ChartFactory.cs

```diff
@@ -44,7 +44,8 @@ public CurveChart CreateChartFor(DataRepository dataRepository, Scalings default
 
          foreach (var valueColumn in dataRepository.AllButBaseGrid().OrderByDescending(x => x.RelatedColumns.Count()))
          {
-            var curve = chart.CreateCurve(valueColumn.BaseGrid, valueColumn, dataRepository.Name, _dimensionFactory);
+            //it's fair to assume that we want the curves to be named after the underlying data
+            var curve = chart.CreateCurve(valueColumn.BaseGrid, valueColumn, valueColumn.Name, _dimensionFactory);
```

Comment:
before the column were set to the name, of the repository??? now name of the column

Reply:
Makes more sense for sure 😆